deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml

```diff
@@ -118,6 +118,7 @@ spec:
             - "--type=rbd"
             - "--controllerserver=true"
             - "--endpoint=$(CSI_ENDPOINT)"
+            - "--csi-addons-endpoint=$(CSI_ADDONS_ENDPOINT)"
```

Comment:
Need same changes in helm chart also

Reply:
@nixpanic and @Madhu-1 do you wish to add the changes for the helm chart in the same PR or different? Otherwise looks good to me.

Reply:
yes, I know! Just being lazy and updated it only here for easy copy/pasting in my test environment.

Reply:
Done now.